modules/send-alerts-to-different-slack-channels.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-with-slack.adoc
+:_module-type: PROCEDURE
+[id="send-alerts-to-different-slack-channels_{context}"]
+= Sending alerts to different Slack channels
+
+[role="_abstract"]
+You can configure {product-title} to send notifications to different Slack channels, so they directly go to the right team.
```

Comment:
s/channels, so they/channels so that they
**so** _conjunction_
Do not use by itself as a conjunction. Use a less ambiguous alternative such as "so that" or "therefore".